### PR TITLE
Danse's Gun Changes 2: With a vengeance

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
This is the same as the previous PR but I have decided to drop the bullet renames because it would be a massively tedious undertaking for relatively little value.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Comment by Geevies** [comment]
Feedback thread here: https://forums.aurorastation.org/topic/14814-feedback-thread-danses-gun-changes
would like to point out the blue stocks not being well received still being here

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -181,18 +181,18 @@
...
 
 /obj/item/projectile/bullet/rifle/a556
 	damage = 30
+	armor_penetration = 20
 
 /obj/item/projectile/bullet/rifle/a556/ap
-	damage = 25
-	armor_penetration = 25
+	damage = 30
+	armor_penetration = 45
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/bullets.dm:195]: this is an excessive amount of AP

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -128,10 +128,10 @@
...
 
 /obj/item/projectile/bullet/pistol/strong
-	damage = 60
-
+	damage = 28
+	armor_penetration = 15
 /obj/item/projectile/bullet/pistol/revolver
-	damage = 45
+	damage = 28
 	armor_penetration = 15
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/bullets.dm:135]: These are two different calibers, why do they do the same damage? Moreover, this seems like an extreme nerf. 30-35 is fine but 28 is far too little.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
The wood stocks should be returned as I don't see enough support for the blue ones.

**Review by Alberyk — APPROVED** [review]

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (TheDanseMacabre) [reply code/modules/projectiles/projectile/bullets.dm:195]: If I understand the code correctly, previously ALL rifle bullets had a base AP of 20. I removed that- and the AP 5.56 is the same as it was before. Correct me if I'm wrong.

File: code/modules/projectiles/projectile/bullets.dm (hunk 2 above)
Comment (TheDanseMacabre) [reply code/modules/projectiles/projectile/bullets.dm:135]: I will buff the pistol/strong bullet up to 30 or 32, as it is supposed to represent .50 AE. To be frank, pistol/revolver desperately needed a nerf. If necessary I'll bring it up to 30.

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/projectiles/projectile/bullets.dm:195]: it isn't additive, you're setting a value to something here - what it was before was 25, not 45

File: code/modules/projectiles/projectile/bullets.dm (hunk 2 above)
Comment (NonQueueingMatt) [reply code/modules/projectiles/projectile/bullets.dm:135]: both need buffs imo

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (TheDanseMacabre) [reply code/modules/projectiles/projectile/bullets.dm:195]: Alright, I'll fix this then

**Comment by TheDanseMacabre** [comment]
I have reverted bluestocks. Kyres has indicated that they may resprite TCFL guns in the future, and if that's the case I don't want to put much more effort into it- especially considering the fact that this PR has been sitting around for so long. If it comes down to it, I'll change it in the future. I don't think black furniture would look right; but green might.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**